README.md

```diff
@@ -80,7 +80,7 @@ You can ask questions by raising an [issue](https://github.com/EddieHubCommunity
 
 - Open the `README.md` file
 
-- **Add your name to the section that matches your Initial in [this list](https://github.com/EddieHubCommunity/hacktoberfest-practice#hacktoberfest-community), make sure that your name in alphabetical order. Then save your changes.**
+- **Add your name to the section that matches your Initial is in [this list](https://github.com/EddieHubCommunity/hacktoberfest-practice#hacktoberfest-community), make sure that your name in alphabetical order. Then save your changes.**
```

Comment:
The previous sentence already has no grammar issues, while unfortunately, yours is.

Reread the sentence and you will understand why.

Reply:
I think , it would be better if the last sentence is changed as "make sure that your name is in alphabetical order"

Reply:
This is indeed a correct suggestion but you didn't commit, change your pr with your new suggestion so we can approve it then